### ISSUE TITLE
wish: Document how to support a bounding box when using the Node.js API

### ISSUE THREAD

**Issue by markstos** [tyrasd/overpass-wizard#22]

**Comment by matkoniecz** [comment]
Have you looked at documentation of how to 

- make http request in Node
- what kind of http request is needed here

?

Also, are you sure it is really about overpass Wizard?

**Comment by markstos** [comment]
Yes, I understand how to make HTTP requests in Node. I maintain `node-config`, `passport-saml`, `node-strava` and other NPM modules.

The documentation provided here
https://github.com/tyrasd/overpass-wizard.

mentions that a `globalBbox` key is supported, but the format required for the `value` to pass to `globalBbox` is not documented.
